Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealitySpeechCommandsProfileInspector.cs

```diff
@@ -64,6 +81,12 @@ public override void OnInspectorGUI()
             }
 
             serializedObject.Update();
+
+            EditorGUILayout.PropertyField(dictationSystemType);
+            EditorGUILayout.PropertyField(speechSystemType);
+            EditorGUILayout.PropertyField(recognizerStart, new GUIContent("Start Behavior"));
```

Comment:
`new GUIContent("Start Behavior")` can be defined as a readonly field so we don't create a new GUIContent every update

Reply:
In fact, if you update the field name to match, you won't even have to make the GUIContent